include/aie-c/TargetModel.h

```diff
@@ -42,6 +42,9 @@ DEFINE_C_API_STRUCT(AieTargetModel, uint64_t);
 
 MLIR_CAPI_EXPORTED AieTargetModel aieGetTargetModel(uint32_t device);
 
+/// Returns the data bus width for the target model.
+MLIR_CAPI_EXPORTED uint32_t aieGetTargetModelDataBusWidth(AieTargetModel targetModel);
```

Comment:
This isn't exactly related to a data bus or stream width, it is the address generation granularity for the DMAs. Perhaps this should be ```aieGetTargetModelAddressGenGranularity```, or something

Reply:
Changed